manager/orchestrator/constraintenforcer/constraint_enforcer.go

```diff
@@ -159,7 +159,8 @@ loop:
 					// restarting the task on another node
 					// (if applicable).
 					t.Status.State = api.TaskStateRejected
-					t.Status.Message = "assigned node no longer meets constraints"
+					t.Status.Message = "task rejected by constraint enforcer"
+					t.Status.Err = "assigned node no longer meets constraints"
```

Comment:
I feel the Message/Err should be swapped here. The error on the task should be related to the task, so "ask rejected by constraint enforcer" seems more relevant.

Also, does it make sense to be more specific around what constraints were failed ?

Reply:
The problem is that we don't display `Err` on the command line interface. `Message` is used to provided better reporting to the user.